.travis.yml

```diff
@@ -64,6 +64,8 @@ matrix:
       env: JOB=tools_unittest TOXENV=py36 HYPOTHESIS_PROFILE=ci SCRIPT=tools/ci/ci_tools_unittest.sh
     - python: 2.7
       env: JOB=resources_unittest TOXENV=py27 SCRIPT=tools/ci/ci_resources_unittest.sh
+    - python: 3.6
```

Comment:
Is this actually worth the performance cost? Ultimately the goal here is to the test the JS rather the Python?